test/katex-spec.js

```diff
@@ -98,10 +100,10 @@ const parseAndSetResult = function(expr, result, settings) {
     } catch (e) {
         result.pass = false;
         if (e instanceof ParseError) {
-            result.message = "'" + expr + "' failed " +
+            result.message = () => "'" + expr + "' failed " +
                 "parsing with error: " + e.message;
```

Comment:
Why did `result.message` change to a callback?

Reply:
This is necessary since some update to Jest: Jest requires all messages to be callable.  Obviously not directly related to this PR, but I ran into this (presumably long-standing) bug when my tests were failing.